volumes-and-secret/volumes-and-secret.py

```diff
@@ -0,0 +1,31 @@
+import kubernetes
+from kfp import components, dsl
+
+@dsl.python_component(
+    name='print_secret_op',
+    base_image="ocdr/dkube-datascience-tf-cpu:v2.0.0-6" 
+)
+def print_secret(user):
+    '''Calculates sum of two arguments'''
+    from kubernetes import client, config
```

Comment:
comment is wrong, lets not DS image for this. use images why has Kubernetes package in it